### PR TITLE
feat: [DEV-12997]: Bundles

### PR DESCRIPTION
**Pull request by Sung96kim**
## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Checklist:

- [ ] Jira story/task is put into PR title
- [ ] Code has been self-reviewed
- [ ] Hard-to-understand areas have been commented
- [ ] Documentation has been updated
- [ ] My changes generate no new warnings
- [ ] Unit tests have been added for base functionality and known edge cases
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: indico/client/request.py [hunk 1]
```diff
@@ -70,6 +70,29 @@ def process_response(self, response: "AnyDict") -> "ResponseType":
         return cast("ResponseType", raw_response)
 
 
+def _parse_nested_response(
```
Comment (jacobmanderson) [inline indico/client/request.py:73]: i swear we've done this like 4 times already
